app/main/posts/modify/post-editor.directive.js

```diff
@@ -113,11 +116,17 @@ function PostEditorController(
             // If attributesToIgnore is set, remove those attributes from set of fields to display
             var attributes = [];
             _.each(attrs, function (attr) {
-                if (!_.contains($scope.attributesToIgnore, attr.key)) {
+                if (attr.type === 'title' || attr.type === 'description') {
+                    if (attr.type === 'title') {
```

Comment:
It might be cleaner (though less efficient) to just do
`_.findWhere(attrs, {type: 'title})` and then process that.. rather than adding many conditions to a loop